connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -209,6 +212,39 @@ private void scanUrlsAndAddPlugins(
             addPlugins(plugins.transformations(), loader);
             transformations.addAll(plugins.transformations());
         }
+
+        loadJdbcDrivers(loader);
+    }
+
+    private void loadJdbcDrivers(final ClassLoader loader) {
+        // Apply here what java.sql.DriverManager does to discover and register classes
+        // implementing the java.sql.Driver interface.
+        AccessController.doPrivileged(
+                new PrivilegedAction<Void>() {
+                    public Void run() {
+                        ServiceLoader<Driver> loadedDrivers = ServiceLoader.load(
+                                Driver.class,
+                                loader
+                        );
+                        Iterator<Driver> driversIterator = loadedDrivers.iterator();
+                        try {
+                            while (driversIterator.hasNext()) {
+                                Driver driver = driversIterator.next();
+                                log.debug(
+                                        "Registered java.sql.Driver: {} to java.sql.DriverManager",
+                                        driver
+                                );
+                            }
+                        } catch (Throwable t) {
+                            log.debug(
```

Comment:
if we're logging this, should we include the exception so you have some idea why this happened?

Reply:
I'm also inclined to err on the side of verbosity although in the original code in `DriverManager` this exception is ignored. Should be a `ClassNotFoundException` meaning that the driver is listed in the metadata but it's class was not found.